.github/workflows/core.yml

```diff
@@ -19,6 +19,35 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Setup Bundler
-      run: ./scripts/setup_bundler.sh
+      run: scripts/setup_bundler.sh
     - name: Build and test
-      run: ./scripts/pod_lib_lint.rb FirebaseCore.podspec --platforms=${{ matrix.target }}
+      run: scripts/pod_lib_lint.rb FirebaseCore.podspec --platforms=${{ matrix.target }}
+
+  core-use-libraries:
+    runs-on: macos-latest
+    if: github.event_name == 'schedule'
+    strategy:
+      matrix:
+        target: [ios, tvos, macos]
```

Comment:
You can have just one definition if you use the matrix to multiply out the combinations:

```yaml
strategy:
  matrix:
    target: [ios, tvos, macos]
    flags: [
      '--skip-tests --use_modular_headers',
      '--skip-tests --use_libraries'
    ]
```
Then below
```yaml
run: scripts/third_party/travis/retry.sh ./scripts/pod_lib_lint.rb FirebaseCore.podspec --platforms=${{ matrix.target }} ${{ matrix.flags }}

Reply:
@paulb777  --use-modular-headers and --use-libraries

Reply:
Yep - thanks! #4853